src/GraphQL/Language/AST/Variable.cs

```diff
@@ -28,5 +28,10 @@ public object Value
         /// Indicates if the variable value has been set.
         /// </summary>
         public bool ValueSpecified { get; private set; }
+
+        /// <summary>
+        /// Indicates if the variable's value is the variable default.
```

Comment:
How does this property correlate with `ValueSpecified`? What combinations are valid and what do they mean?

Reply:
Presently, it’s “set” either explicitly or via the default specified in the query. So if IsDefault is false then ValueSpecified is false.

Reply:
IsDefault is if the variable wasn’t set in the inputs but has a configured default value in the query

Reply:
Although there is information contained within IsDefault, it should be completely invisible to the execution (as it was specified by the requester) which is why I had not previously included it.

Reply:
> IsDefault is if the variable wasn’t set in the inputs but has a configured default value in the query

is _true_ ? Also worth to add in comments.

Reply:
```suggestion
        /// Indicates if the variable's value is the variable's configured default value.
```

Reply:
OK